presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestMinimalFunctionality.java

```diff
@@ -136,4 +138,17 @@ public void testTopicHasData()
 
         assertEquals(result, expected);
     }
+
+    @Test
+    public void testTopicMetadata()
+    {
+        createMessages(topicName, 1);
+
+        MaterializedResult result = queryRunner.execute("SELECT _timestamp from " + topicName);
+        MaterializedRow row = result.getMaterializedRows().get(0);
+        long messageCreationTimestamp = (Long) row.getField(0);
+
+        assertTrue(result.getMaterializedRows().size() > 0);
```

Comment:
I would move this assertion to L148 (before accessing the row) 
```suggestion
        assertEquals(result.getRowCount(), 1);
```